src/localization/online_localizer/online_localizer.cpp

```diff
@@ -238,23 +204,19 @@ bool OnlineLocalizer::nodeWorthExpanding(const Node &node) const {
     return true;
   }
   // if it the current best hypothesis
-  if (node == _currentBestHyp) {
+  if (node == currentBestHyp_) {
     return true;
   }
 
-  int row_dist = _currentBestHyp.quId - node.quId;
-  if (row_dist < 0) {
-    printf(
-        "[ERROR][INTERNAL][OnlineLocalizer] Trying to expand a node further in "
-        "future %d than current best hypothesis %d.\n",
-        node.quId, _currentBestHyp.quId);
-    exit(EXIT_FAILURE);
-  }
+  int row_dist = currentBestHyp_.quId - node.quId;
```

Comment:
This can be const. It is not important for performance but it is a nice habit to have, so I will try to point out other places like this too.